### PR TITLE
New endpoints structure

### PR DESCRIPTION
**Pull request by trojkat**
More RESTful URLs.
new-endpoints

### CODE REVIEW

**Comment by joeldalley** [comment]
Looks good. I am seeing a conflict warning, though.

**Comment by trojkat** [comment]
I will create a new PR directly to the ExpediaGroup/icf repository.

File: docs/openapi_v1.0.0-alpha.yaml [hunk 1]
```diff
@@ -202,18 +182,49 @@ paths:
...
+      description: |-
+        **WARNING: this endpoint may live under a different domain, path or both depending on the supplier endpoint URL returned by the GET /suppliers.**
+
+        Returns a list of dates on which tickets are available.
+        Only dates that have availability SHALL BE returned.
+        A minimum range of 31 days MUST BE supported.
+
+        This endpoint proved a relatively lightweight way to show a calendar in a user interface containing a one month calendar with the days where there is no availability greyed out.
+
+  /suppliers/{supplierId}/bookings:
```
[the remark this answers is not in the document]
Comment (trojkat) [reply docs/openapi_v1.0.0-alpha.yaml]: This is an endpoint for creating a reservation, the `/suppliers/{supplierId}/bookings/{uuid}` returns details of an existing booking object.

**Comment by trojkat** [comment]
The new PR: https://github.com/ExpediaGroup/icf/pull/3

**Pull request closed without merging**